.chglog/config.yml

```diff
@@ -6,12 +6,13 @@ info:
   repository_url: https://github.com/logto-io/go
 options:
   commits:
-    # filters:
-    #   Type:
-    #     - feat
-    #     - fix
-    #     - perf
-    #     - refactor
+    filters:
+      Type:
+        - feat
+        - fix
+        - perf
+        - refactor
+        - chore
```

Comment:
chore should be filtered as well? not sure though

Reply:
Yes, the chore stuff seems redundant.